### PR TITLE
Finishing Mobile version

### PR DESCRIPTION
**Pull request by MarkYegon7**
In this project, I was able to:

- Use the correct GitHub flow principles.
- Use Flexbox and CSS Grid to align elements.
- Use linters.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,6 +36,191 @@ <h2 class="lets-connect">LET'S CONNECT</h2>
...
+
+                <h3 class="connect-1">LET'S CONNECT</h3>
+
+                <ul class="icons-1">
+                    <li><img src="images/Vector (4).svg" class="icons-2"></li>
+                    <li><img src="images/Group.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (5).svg" class="icons-2"></li>
+                    <li><img src="images/Vector.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (3).svg" class="icons-2"></li>
+                </ul>
```
Comment (Qoosim) [inline index.html:178]: - [ ] It is required that the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag). Kindly use <a> tag to add links to your social icons.

- [ ] Kindly reduce the left-margin of all the icons in order to perfectly match the figma design

### Your Design
your-about

### Figma Design
temp-about

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @MarkYegon7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Fair README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (MarkYegon7) [reply index.html]: Implemented the said changes

File: index.html [hunk 2]
```diff
@@ -36,6 +36,191 @@ <h2 class="lets-connect">LET'S CONNECT</h2>
...
+
+                <h3 class="connect-1">LET'S CONNECT</h3>
+
+                <ul class="icons-1">
+                    <li><a href="#"><img src="images/Vector (4).svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Group.svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Vector (5).svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Vector.svg" class="icons-2"></a></li>
+                    <li><a href="#"><img src="images/Vector (3).svg" class="icons-2"></a></li>
+                </ul>
```
Comment (souad988) [inline index.html:178]: [OPTIONAL] Good job by making the changes requested by the previous reviewer 👍🏼  and I would ask you please to reduce the spacing between this list items to make it match the design in figma 

image (30)

**Review by souad988 — APPROVED** [review]
Hi @MarkYegon7 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
Good PR description
Good communication with the previous reviewer

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**